### PR TITLE
Release/cognitiveservices/clu 2021 11 01 preview

### PR DESCRIPTION
**Pull request by fxs130430**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [ ] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  4. If updating an existing version, please select the specific langauge SDKs and CLIs that must be refreshed after the swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No refresh required for updates in this PR

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist 

> **Applicability**: :warning: 
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs 
> - Adding new properties 
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [ ] Check this box if any of the following apply to the PR so that label "WaitForARMFeedback" will be added automatically to begin ARM API Review. Failure to comply may result in delays to the manifest.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you are using OpenAPIHub to initialize the PR for adding a new version. More details, refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [ ] Removing API(s) in a stable version
- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Review by heaths — APPROVED** [review]
Verified via previously-reviewed commits.

**Comment by heaths** [comment]
A few validation errors:

1. The spelling and linting issues should be straight forward to fix, but let me know if I can help.
2. @JeffreyRichter the [BreakingChange](https://github.com/Azure/azure-rest-api-specs/pull/17507/checks?check_run_id=4947334881) issue is actually a tool failure. If I use the OpenAPI Hub diffing tool against both the source and merged swaggers using the commit ID for both, the [tool reports](https://portal.azure-devex-tools.com/tools/diff/eyJzb3VyY2VVcmwiOiJodHRwczovL2dpdGh1Yi5jb20vQXp1cmUvYXp1cmUtcmVzdC1hcGktc3BlY3MvYmxvYi9kZjdiY2JlZmIyOTkzNDg0NDc2YTQ5NmVlYTYwMGMzZGM5Nzc3YzY1L3NwZWNpZmljYXRpb24vY29nbml0aXZlc2VydmljZXMvZGF0YS1wbGFuZS9MYW5ndWFnZS9wcmV2aWV3LzIwMjEtMTEtMDEtcHJldmlldy9hbmFseXplY29udmVyc2F0aW9ucy5qc29uIiwidGFyZ2V0VXJsIjoiaHR0cHM6Ly9naXRodWIuY29tL0F6dXJlL2F6dXJlLXJlc3QtYXBpLXNwZWNzL2Jsb2IvOGYxMDZhNWZmYjY4MzdlYTJiM2I3OWNjOTZlZTVkZjkyM2Y3ZWNjOS9zcGVjaWZpY2F0aW9uL2NvZ25pdGl2ZXNlcnZpY2VzL2RhdGEtcGxhbmUvTGFuZ3VhZ2UvcHJldmlldy8yMDIxLTExLTAxLXByZXZpZXcvYW5hbHl6ZWNvbnZlcnNhdGlvbnMuanNvbiJ9) no differences - as expected, as you can see by the commits in this PR that was rebased against main and came from [this feature branch](https://github.com/Azure/azure-rest-api-specs/tree/release/cognitiveservices/clu-2021-11-01-preview).
3. The .NET (and likely Python) error are from a bad merge, I think. [The check](https://github.com/Azure/azure-rest-api-specs/pull/17507/checks?check_run_id=4947345673) is showing duplicate values:
```
cmderr	[Autorest]   Error: '$.definitions.ErrorCode.enum' has incompatible values (---
cmderr	[Autorest] - InvalidRequest
cmderr	[Autorest] - InvalidArgument
cmderr	[Autorest] - Unauthorized
cmderr	[Autorest] - Forbidden
cmderr	[Autorest] - NotFound
cmderr	[Autorest] - TooManyRequests
cmderr	[Autorest] - InternalServerError
cmderr	[Autorest] - ServiceUnavailable
cmderr	[Autorest] , ---
cmderr	[Autorest] - InvalidRequest
cmderr	[Autorest] - InvalidArgument
cmderr	[Autorest] - Unauthorized
cmderr	[Autorest] - Forbidden
cmderr	[Autorest] - NotFound
cmderr	[Autorest] - ProjectNotFound
cmderr	[Autorest] - OperationNotFound
cmderr	[Autorest] - AzureCognitiveSearchNotFound
cmderr	[Autorest] - AzureCognitiveSearchIndexNotFound
cmderr	[Autorest] - TooManyRequests
cmderr	[Autorest] - AzureCognitiveSearchThrottling
cmderr	[Autorest] - AzureCognitiveSearchIndexLimitReached
cmderr	[Autorest] - InternalServerError
cmderr	[Autorest] - ServiceUnavailable
cmderr	[Autorest] ).
cmdout	[Autorest] Process() cancelled due to exception : '$.definitions.ErrorCode.enum' has incompatible values (---
cmdout	[Autorest] - InternalServerError
cmdout	[Autorest] - InternalServerError
```

File: specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/common.json
Comment (heaths) [inline specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/common.json:103]: Make sure spacing is correct. Not sure for this repo, but some validation will make sure formatting is correct. The spacing looks off here, though it may just be GitHub's rendering.

**Review by heaths — COMMENTED** [review]
I would also take the opportunity to alphabetize these, though just a suggestion. If people keep doing that, would make reviews a little easier.

File: specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/common.json
Comment (fxs130430) [reply specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/common.json]: Oops, fixed.

**Comment by heaths** [comment]
@JeffreyRichter this is only breaking because they are adding a new swagger for a different "sub-service" when Text Analytics already happens to be in this version and in readme.md. This is a problem I'm also discussing with @johanste later today (and how to remedy) it. Since we produce different SDKs, we either need different swaggers and target those directly - ignoring readme.md (which then validation should as well), or splitting them into different tags - or every Cognitive service gets a unique api-version and we have to manually review changes to common patterns across the set of api-versions.

**Comment by JeffreyRichter** [comment]
So, do you want me to do anything here or you just need more time to sort things out?

**Comment by heaths** [comment]
> So, do you want me to do anything here or you just need more time to sort things out?

Could you approve this. Text Analytics is also likely to pull out of 2022-03-01-preview anyway. Talking with @abhahn yesterday, I hope the need to do this sort of work in a feature branch is something we'll see going forward.

**Review by heaths — APPROVED** [review]

**Review by heaths — APPROVED** [review]

**Comment by heaths** [comment]
@JeffreyRichter the [breaking change](https://github.com/Azure/azure-rest-api-specs/pull/17507/checks?check_run_id=5309763946) is because TA had already checked into main with a preview version, and CLU is trying to merge now. While these appear in the same folder, they do create separate SDKs, as will always be the case with the Language RP (created as part of unification). Long term we need to consider how to improve this situation of multiple SDKs for a single RP (or maybe splitting RPs is the answer), but can you approve this for now?

File: CODEOWNERS
Comment (weidongxu-microsoft) [inline CODEOWNERS:16]: Would you please add team `@Azure/api-stewardship-board` as code owner, as well?

I didn't find the linked issue of API stewardship review. Let me know if there is one (would like to see previous meeting minutes there).

Comment (fxs130430) [reply CODEOWNERS]: I'm not aware of the issue as I recently took over.

Comment (weidongxu-microsoft) [reply CODEOWNERS]: We would like to have some context of the change to continue the review (either you would need to book a meeting, or let me or Ted know whether this PR is a continuation from a preview review meeting).

Ref https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/271/Azure-REST-API-Stewardship

Comment (fxs130430) [reply CODEOWNERS]: Let me clarify: Currently, we have two open PR: 
1 - clu 2021-11-01-preview (this one) which has already gone through the API board review process, but for some reason, the PR was never merged into the main. Unfortunately, the engineer who drove this effort is on parental leave. @heaths reached out to get this done and I was assigned to work with him. That's why I do not have the full knowledge of board meeting minutes and the issue.

2 - clu 2022-03-01-preview which is under review by the API board. We presented this version on Feb 8th and you can find the design document, minutes as well as the recorded meeting. During the meeting I provided an overview of our service as well as the new capabilities.

https://github.com/Azure/azure-rest-api-specs/pull/17504
https://github.com/Azure/azure-rest-api-specs/issues/17505

I'll be happy to schedule a meeting to answer any questions that you might have.

Comment (weidongxu-microsoft) [reply CODEOWNERS]: Thanks Farhad.
I was trying to find the meeting minutes (which should be similar to the one for 2022-03-01-preview at https://github.com/Azure/azure-rest-api-specs/issues/17505), so that I know what to focus on the change requests there. If you can find it, let us know (not necessary an issue, just notes mentioning what was discussed in the review meeting).
If you cannot find it, we might had to review the whole swagger offline (which could take more time).

File: specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/common.json
Comment (weidongxu-microsoft) [inline specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/common.json:18]: Based on feedback to 2022-03-01-preview, `error` should be required.

File: specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json
Comment (weidongxu-microsoft) [inline specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json:556]: Can `body` really be nullable?

File: specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json
Comment (weidongxu-microsoft) [inline specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json:368]: nit, this "confidenceScore" is double, while others is float. Is it intended?

File: CODEOWNERS
Comment (heaths) [reply CODEOWNERS]: It is deployed. The problem with Cognitive Services in this repo is, unfortunately, with the unified approach they share an RP but are actually separate SDKs. The current validation doesn't really support this and we may need to think having a single RP and what ramifications that will have if/when we do. But the idea was to keep a set of common definitions for parameters and models (common.json, which TA in 2021-11-01-preview doesn't use but will in newer versions) to help facilitate a common feel across Language pillar services.

So, in effect, there's really no breaking change here, if that's the concern.

File: specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json
Comment (fxs130430) [reply specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json]: no, could be a typo. I changed it to float

File: specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json
Comment (fxs130430) [reply specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json]: no, most likely a typo. I removed it.

File: specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/common.json
Comment (fxs130430) [reply specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/common.json:18]: good catch. I made it required.

**Review by weidongxu-microsoft — APPROVED** [review]

**Comment by fxs130430** [comment]
@heaths is there any action item here for me?

**Comment by heaths** [comment]
@lmazuel, @mayurid I've kicked the pipelines that failed but didn't report back to GitHub, but the issues appear benign or unrelated. Can we get this merged so we can get a proper CLU SDK out the door?

File: CODEOWNERS
Comment (heaths) [reply CODEOWNERS]: @weidongxu-microsoft I'm on the SDK team and have been working on the SDK along with others. The service itself did go through an initial review quite a while back and has been under some development through the fall.

**Comment by weidongxu-microsoft** [comment]
@fxs130430 
If possible see if Mike's comment on https://github.com/Azure/azure-rest-api-specs/pull/17504 also apply here?

@mikekistler 
Here is the 2021-11-01-preview api-version on CLU which is deployed. It is a prior version of the 2022-03-01-preview api-version you've reviewed.
Do you see any other change that the PR need to address?

**Comment by heaths** [comment]
@lmazuel here's another where status wasn't reported back - presumably because both checks had runtime failures but, before that, showed no further issues.

File: specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json
Comment (mikekistler) [inline specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json:162]: You should avoid the use of `x-nullable: true`. Properties with no value should simply be omitted from the payload.

File: specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json
Comment (mikekistler) [inline specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json:390]: Avoid the use of `x-nullable: true`. Properties with no value should simply be omitted from the payload.

File: specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json
Comment (mikekistler) [inline specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json:543]: Avoid the use of `x-nullable: true`. Properties with no value should simply be omitted from the payload.

File: specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json
Comment (mikekistler) [inline specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json:201]: Property names should be lowerCamelCase.

File: specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json
Comment (mikekistler) [inline specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json:213]: Property names should be lowerCamelCase.

**Review by mikekistler — APPROVED** [review]
Found and flagged a few minor style issues.  Otherwise LGTM.

File: specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json
Comment (heaths) [reply specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json:162]: I want to clarify this, though. While I agree with @mikekistler, I know from experience that quite a few implements don't or can't (like some using common OData libraries), in which case you should keep `"x-nullable": true` because, if you don't, code generation can be wrong.

Comment (johanste) [reply specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json:162]: If the service is using a library that is unable to omit the value, we should have a conversation on how to address/fix that. A nullable optional property is very unfortunate to have in the Service API.

Comment (weidongxu-microsoft) [reply specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json:162]: I read .NET? :-)

If service lib does return `"property": null`, and .NET does require `x-nullable` to take a `null` property, we seems not able to remove it?

Comment (heaths) [reply specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json:162]: More problematic for the default tag is that it won't work with the current cognitiveservices/Language RP. Despite being in a single RP meant for a Language pillar unification including - originally, but no longer - release schedules, each swagger (at least by the first dash-delimited file name term) is actually a distinct SDK. They will ship on different schedules, though we want to try to maintain fewer api-versions across them. So either we need to rethink how we tag (default or otherwise) and validation such swaggers, or we need to separate them into separate RPs - distinct from the old ones, since these are actually new and contain potentially many breaking changes.

Comment (weidongxu-microsoft) [reply specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json:162]: Agree, Heath. Josehpine Qiu is also reviewing the readme structure, though her main attention is for MSDocs. SDK with Renhe & Laurent is also reviewing the same thing for data-plane SDK purpose.

Current validation structure mostly works fine for management (though still corner case for resources and compute), but it very likely not working good for data-plane.

Comment (ChongTang) [reply specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json:162]: Actually in the service's response, there is no null values will be returned. We ignore all null values in the response.

Weidong mentioned this: ”I've added SignedOff label, as this is more on SDK than API.“, and this is the actual case.

Can we make a decision on this right now? We will take the decision in our next version.

Comment (weidongxu-microsoft) [reply specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json:162]: If service does not return `null`, there seems to be no reason to have these `x-nullable`? (I seems seeing this being fixed in https://github.com/Azure/autorest.java/pull/1377 on Mike's recommendation but that PR get closed)

As we already have 2 approvals, we can merge when requested (probably had to bypass Avocado and LintDiff).

File: CODEOWNERS
Comment (mikekistler) [reply CODEOWNERS]: I don't have any objections to merging. We can address the `x-nullable` concern in a future PR.

File: specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json
Comment (weidongxu-microsoft) [reply specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json:213]: @fxs130430
Does service really use this field name?

As guideline, it should be camel case.
https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#json-resource-schema--field-mutability

If service does return this, you cannot fix this in the deployed api-version as it breaks. But please discuss whether to fix in in 2022-03-01-preview

**Comment by ChongTang** [comment]
Hi @weidongxu-microsoft and @jianyexi . All validations succeeded previsouly but the last trigger failed the Avocado check. The error message says that there are APIs missing in the default tag. However, we do not have a default tag. This check was added 12 days ago in Avocado in this commit: https://github.com/Azure/avocado/commit/024ebd33856a264cd609814e3378781eff7cb84a

The instruction is not so helpful. Do you have an example of how to create a default tag and how to include all APIs in the readme file? Thanks! We have another PR which has the same issue.

**Comment by tjprescott** [comment]
@ChongTang I think the issue is that your 2022-02-01-preview tag does not include `analyzeconversations.json`. Thus anything generated with the new tag will not have any of these APIs:

image

**Comment by ChongTang** [comment]
> 

Thanks Travis! This is on purpose. We have multiple services under the same "Language" org. These services have different release cadence. Is there a way to skip this "Missing API in the default tag" given our situation?

**Comment by tjprescott** [comment]
@ChongTang no, because if you generate an SDK with this tag, it will no longer have the analyze conversation APIs, which would be massively breaking.

**Comment by ChongTang** [comment]
> @ChongTang no, because if you generate an SDK with this tag, it will no longer have the analyze conversation APIs, which would be massively breaking.

I see. Thanks Travis. Then we need to think about what we should do. Thanks for your help.

**Comment by weidongxu-microsoft** [comment]
@ChongTang @tjprescott
We should be able to skip this when merge. (we are good at all significant comments are addressed).

The `MISSING_APIS_IN_DEFAULT_TAG` actually does not mean much in data-plane, when many service doing multiple SDKs in a single service.
This is useful for management, as usually one service/RP need to release all the JSONs into single SDK.

I will notify swagger team on this.

LintDiff is due to bug on tooling, they are fixing, but we should be able to skip if neccessary.

**Comment by ChongTang** [comment]
> @ChongTang @tjprescott We should be able to skip Avocado when merge. (we are good as all significant comments have been addressed).
> 
> The `MISSING_APIS_IN_DEFAULT_TAG` actually does not mean much in data-plane, when many service doing multiple SDKs in a single service, and the README only have one default tag. This is useful for management, as usually one service/RP need to release all the JSONs into single SDK.
> 
> I will notify swagger team on this.
> 
> LintDiff is due to bug on tooling, they are fixing, but we should be able to skip if necessary (I can find a good LintDiff in older commits so I assume this PR is good https://github.com/Azure/azure-rest-api-specs/runs/4974666474).

Thanks Weidong! Can you help us skip the `MISSING_APIS_IN_DEFAULT_TAG` check?

We have another PR which has the same problem. Could you help us skip the `MISSING_APIS_IN_DEFAULT_TAG` check as well? Thanks! Here is the link: https://github.com/Azure/azure-rest-api-specs/pull/18067

**Comment by heaths** [comment]
@lmazuel @kurtzeborn could you help us merge this given the remaining validation errors are irrelevant to the changes in this PR?

**Comment by tjprescott** [comment]
@heaths so are the endpoints described in `analyzeconversation.json` _not_ part of the textanalytics library? If so, I don't see how you can exclude that from 2022-02-01-preview without it producing a broken SDKs that is missing a bunch of APIs.

**Comment by ChongTang** [comment]
> @heaths so are the endpoints described in `analyzeconversation.json` _not_ part of the textanalytics library? If so, I don't see how you can exclude that from 2022-02-01-preview without it producing a broken SDKs that is missing a bunch of APIs.

Hi @tjprescott. Exactly. analyzeconversation.json contains API(s) from CLU only, Text Analytics' APIs are in textanalytics.json. These two services have different release cadence. CLU doesn't have a 2022-02-01-preview version.

**Review by tjprescott — CHANGES_REQUESTED** [review]
We have a meeting scheduled tomorrow to discuss--so holding on this PR until then.

**Review by tjprescott — APPROVED** [review]

**Comment by heaths** [comment]
@kurtzeborn @mikeharder the branch policy has the same issue we saw last Thursday where the pipeline name was changed but the branch policy is still waiting for the old name:

image

The pipelines are called merely "LintDiff" and "LintRPaaS".

**Comment by weshaggard** [comment]
@akning-ms we need to get these branch policies cleaned up as I think a lot more branches are going to be hitting this issue.

**Comment by weshaggard** [comment]
>The pipelines are called merely "LintDiff" and "LintRPaaS".

@heaths the required checks here are actually running and named correctly  (not the names are not matching what is in the pipeline because the bot reports the statuses here). It looks like bot has just not reported back their status yet.

**Comment by heaths** [comment]
@weshaggard here's what I see in DevOps:

image

(Failures in Avocado and BreakingChanges were expected)

**Comment by weshaggard** [comment]
@heaths Yes I understand but those job names don't map directly to the status checks in this repo. They have setup a level of indirection and the status checks come from the bot not DevOps. @ruowan can probably explain better then I can on how this setup. However the status checks here are in a pending state so it isn't an issue with the required checks it is an issue that the status checks haven't been reported back as completed.

**Comment by weshaggard** [comment]
After talking with some folks this PR is ready to merge so I'm going to override the checks and merge.

**Comment by heaths** [comment]
@weshaggard 

> They have setup a level of indirection and the status checks come from the bot not DevOps.

Ah, I didn't realize. Thanks for mentioning that.

**Pull request merged**
